kotlin/readme.md

```diff
@@ -0,0 +1,45 @@
+# INJI VCI Client
+
+## Features
+
+- Request credential to Credential issuer
+
+## Installation
+
+TODO: add installation guide
+
+## APIs
+
+##### Request Credential
+
+Initiates a credential request to the credential issuer and returns back the credential.
```

Comment:
Request for credential from the providers (credential issuer), and receive the credential back.